src/core/test/fuzzing/src/test/scala/FuzzingTest.scala

```diff
@@ -67,6 +68,7 @@ class FuzzingTest extends TestBase {
     val exemptions: Set[String] = Set(
       "com.microsoft.ml.spark.TrainClassifier",
       "com.microsoft.ml.spark.TrainRegressor",
+      "com.microsoft.ml.spark.TuneHyperparameters",
```

Comment:
THis one too

Reply:
removed

Reply:
Still here lol

Reply:
(as discussed I re-added this back for now due to a different issue in fuzzing)